### DISCUSSION TITLE
Access dipoles from LAMMPS

### DISCUSSION THREAD

**Discussion by ppegolo** [deepmodeling/deepmd-kit#2407] [Q&A]
Hi all,

Is there any way to compute on-the-fly the dipoles from a deepmd dipole model of the atoms in a LAMMPS simulations carried out with an energy model? I.e., I would like to use both an energy and a dipole model at the same time in LAMMPS without using DPLR.
Is this is not implemented yet, can anybody hint at where I should look in the code to do so?

Thanks,
Paolo

**Comment by njzjz** [comment]
Please check [`compute deeptensor/atom`](https://docs.deepmodeling.com/projects/deepmd/en/master/third-party/lammps-command.html#compute-tensorial-properties).
